sros2/test/policies/permissions/single_context/permissions.xml

```diff
@@ -0,0 +1,195 @@
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.omg.org/spec/DDS-SECURITY/20170901/omg_shared_ca_permissions.xsd">
```

Comment:
Why are these files committed? Will they be used in the tests?
Or is it just to show what the resulting permission files would be ?

Reply:
I checked them in, as it was helpful to track what alterations in the final permission file would occur when modifying the template used for various policy formations: e.g.  c4b20e3f1bf8ebaaa9902be078ae40b6e859efcc

It would also be nice to also just reuse there for unit testing for further transforms checks.